docs/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,68 @@
+Thanks for filing a bug! We provide an issue template for you to make sure
```

Comment:
If you surround this block with `<!-- ` and `-->` then even if they accidentally leave it in then it will not be visible anyway.  :-)